README.md

```diff
@@ -22,3 +22,36 @@ There is also a demo application:
 
     cd tensorflow-mnist
     stack --docker --docker-image=$IMAGE_NAME build --exec Main
+
+## Build on Mac OS X
+
+The following instructions were verified with Mac OS X El Capitan.
+
+- Install dependencies via [Homebrew](http://brew.sh):
+
+        brew install swig
+        brew install bazel
+
+- Build the TensorFlow library and install it on your machine:
+
+        cd third_party/tensorflow
+        ./configure  # Choose the defaults when prompted
```

Comment:
That script just calls ./configure, and its comments suggest it's a hack specific to Docker.  So I think `./configure` is more appropriate here.  (It's also what the other language bindings recommend.)